classes/book.rb

```diff
@@ -0,0 +1,10 @@
+# class for books
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
```

Comment:
- [ ] Also, in book.rb there should be a method for adding rental kindly look through to above suggestion. But here parameters would be like that `(person, date)`. Kindly add it as well. Thank you :)